### ISSUE TITLE
[Bug]: Galaxy-Flasher crashes when the system language isn't English

### ISSUE THREAD

**Issue by emjaaa** [ethical-haquer/Galaxy-Flasher#22]
### What happened?

I compiled Galaxy-Flasher as a flatpak with your tutorial. It worked fine after getting the right dependencies. I saw the desktop-entry for Galaxy-Flasher and it showed up as installed but I was not able to run it. Even when running it in the terminal with `flatpak run com.ethicalhaquer.galaxyflasher` there was no output at all. 

I am using **debian testing** with **only German locales installed**.

Later I was able to get an error message which is attached below. Changing my system language to **English** solved the problem.

Maybe something is wrong with the language fallback.

Thank you for your beautiful flashing tool. It is just super nice to use!

### Galaxy Flasher version

Alpha v0.5.1

### Galaxy Flasher's log output

```shell
$ flatpak run com.ethicalhaquer.galaxyflasher 

Traceback (most recent call last):
  File "/app/./galaxy-flasher", line 1532, in on_activate
    self.win = MainWindow(application=app)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/./galaxy-flasher", line 74, in __init__
    with open(locale_file) as json_string:
         ^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] File or directory not found: '/app/locales/de.json'
^C
$ Traceback (most recent call last):
  File "/app/./galaxy-flasher", line 1538, in <module>
    app.run(sys.argv)
  File "/usr/lib/python3.11/site-packages/gi/overrides/Gio.py", line 40, in run
    with register_sigint_fallback(self.quit):
  File "/usr/lib/python3.11/contextlib.py", line 144, in __exit__
    next(self.gen)
  File "/usr/lib/python3.11/site-packages/gi/_ossighelper.py", line 237, in register_sigint_fallback
    signal.default_int_handler(signal.SIGINT, None)
KeyboardInterrupt
^C
```

**Comment by emjaaa** [comment]
I retested and edited the first message accordingly. I was also able to reproduce this error on another system with Fedora 40. 
I just changed my system language to German and Galaxy-Flasher did not work anymore. Changing it back to English solved it again.

**Comment by justaCasualCoder** [comment]
Galaxy Flasher has not been translated to German yet. That's what's causing the error.

**Comment by ethical-haquer** [comment]
@emjaaa 
Thanks for the kind words!
This is the code that loads the strings (text):
```
locale_file = f"{swd}/locales/{lang}.json"
```
Since Galaxy Flasher only has English translations currently, it will basically refuse to run if the system language isn't English. 🤦 Instead, it should just fall back to using English. I'll be sure to fix this in the next release, sorry about this!

**Comment by ethical-haquer** [comment]
Fixed in v0.6.0.
